docs/shared/console.md

```diff
@@ -87,13 +87,13 @@ You can also launch other common Nx commands with the options listed out in the
 
 #### Projects
 
-Clicking on the name of any project will navigate to that project's definition in the `workspace.json` (or `angular.json`) file. Clicking on the name of any builder command will navigate to that builder command's definition in the `workspace.json` (or `angular.json`) file.
+Clicking on the name of any project will navigate to that project's definition in the `workspace.json` (or `angular.json`) file. Clicking on the name of any executor command will navigate to that executor command's definition in the `workspace.json` (or `angular.json`) file.
 
-Clicking the [refresh-light.svg] icon next to the `PROJECTS` header will repopulate the Projects pane from the `workspace.json` (or `angular.json`) file.
+Clicking the ![refresh-light.svg](./refresh-light.svg) icon next to the `PROJECTS` header will repopulate the Projects pane from the `workspace.json` (or `angular.json`) file.
 
-Clicking the [folder-light.svg] icon next to a project will reveal that project's folder in the VSCode Explorer pane.
+Clicking the ![folder-light.svg](./folder-light.svg) icon next to a project will reveal that project's folder in the VSCode Explorer pane.
```

Comment:
This doesn't work, but neither did what was there before.

What's the correct approach for this?

Reply:
You may need an absolute path here to `/shared/folder-light.svg`. I think that images not working is an existing issue for PR previews.

Reply:
Seems like it also isn't working. Even locally. I'm wondering if it's because it's an `svg` and not a `png` or similar image type.

Reply:
docs check is saying `/shared/{filename}.svg` are invalid links